packages/compose/src/hooks/use-resize-observer/index.ts

```diff
@@ -0,0 +1,119 @@
+/**
+ * WordPress dependencies
+ */
+import { useRef } from '@wordpress/element';
+/**
+ * Internal dependencies
+ */
+import useEvent from '../use-event';
+import type { ObservedSize } from './_legacy';
+import _useLegacyResizeObserver from './_legacy';
```

Comment:
We could remove the underscore from the `_legacy` folder name. It's a regular source dir that will stay with us for a long time.

Reply:
Done https://github.com/WordPress/gutenberg/pull/65389/commits/160cb5b0a42e7570d7c150d7dad685f673b42dfc